src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -4,16 +4,11 @@ set(SOURCES
     SymSgdNative.cpp
 )
 
-if(WIN32)
-    find_library(MKL_LIBRARY MklImports HINTS ${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/0.0.0.5/runtimes/win-x64/native)
-else()
+file(TO_NATIVE_PATH "${MKL_LIB_PATH}" MKL_LIB_PATH)
+find_library(MKL_LIBRARY MklImports HINTS ${MKL_LIB_PATH})
+if(CMAKE_SYSTEM_NAME STREQUAL Linux OR CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
This should be not `!WIN32`  (whatever the NOT is in CMAKE).   The reason is because there are other systems that aren't linux or mac (say FreeBSD), and we would need to change this logic here if we ever tried supporting those systems.